pkg/javadb/client.go

```diff
@@ -107,7 +107,7 @@ func (u *Updater) downloadDB(ctx context.Context) error {
 		return xerrors.Errorf("failed to download vulnerability DB: %w", err)
```

Comment:
@nikpivkin update this comment, please.

Reply:
@DmitriyLewen Done https://github.com/aquasecurity/trivy/pull/7642/commits/3575c1819e5d3d82d197582c1514e379bfc4303d